packages/storage-ui/cypress/support/page-objects/bucketsPage.ts

```diff
@@ -24,19 +25,25 @@ export const bucketsPage = {
   deleteBucketMenuOption: () => cy.get("[data-cy=menu-delete-bucket]"),
 
   // helpers and convenience functions
-  createBucket(bucketName: string, bucketType: FILE_SYSTEM_TYPES) {
+  awaitBucketRefresh() {
+    cy.intercept("GET", "**/buckets/summary").as("refresh")
+    cy.wait("@refresh")
+  },
+
+  createBucket(bucketName: string, bucketType: BucketType) {
```

Comment:
You could do this 

```suggestion
  createBucket(bucketName: string, bucketType: "ipfs" | "chainsafe") {
```
and then no need to have `type BucketType = "ipfs" | "chainsafe"` declared towards the top of the file.

As there are only two types of bucket you also get rid of the switch for with a simpler if / else statement

Reply:
make sense to have it as a parameter because is only used there! and... yeah as well make sense that can be solved with if/else because is only 2 types of file system